src/wblib/template.qmd

```diff
@@ -113,7 +103,15 @@ title: "PERCUSION Weather briefing"
 ## Precipitation Forecast
 ![]({{< meta plots.internal.precip.108h >}})
 
-
+##
+:::: {.columns}
```

Comment:
What does this and the following line do? Column width = 30% seems to have been set earlier in the document already (line 13 + 14).

Reply:
Hi Luca. Thanks! Those add columns to the presentations, see https://quarto.org/docs/presentations/#multiple-columns. The ones after line 13 are independent from those after line 107, notice the 4 colons used to start the columns context. These come from a previous commit and where just moved to different places for this pull request.